wezterm-gui/src/overlay/selector.rs

```diff
@@ -178,25 +189,46 @@ impl SelectorState {
     }
 
     fn run_loop(&mut self, term: &mut TermWizTerminal) -> anyhow::Result<()> {
+        let max_items = self.max_items;
+        let alphabet = self.args.alphabet.to_lowercase();
```

Comment:
Since compute_labels_for_alphabet starts like this:

```rust
pub fn compute_labels_for_alphabet(alphabet: &str, num_matches: usize) -> Vec<String> {
    let alphabet = alphabet
        .chars()
        .map(|c| c.to_lowercase().to_string())
        .collect::<Vec<String>>();
```

I think we need to make sure that we work with lower case for the alphabet (e.g. when checking if alphabet.contains("j")).  But I have moved it to the construction of SelectorState now.

Reply:
I changed it now, so you can use uppercase letters. (Note: To do this I made a tiny edit to compute_labels_for_alphabet).